<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>683</width>
    <height>384</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>683</width>
    <height>384</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PySide2 QFileDialog().</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../images/icons/icon.png</normaloff>../../../../images/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Clique em um dos botões</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_open_file">
      <property name="text">
       <string>Selecionar arquivo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_open_files">
      <property name="text">
       <string>Selecionar arquivos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_open_dir">
      <property name="text">
       <string>Selecionar um diretório</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_save_file">
      <property name="text">
       <string>Salvar um arquivo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
